single-sign-on/faqs.md

```diff
@@ -5,83 +5,215 @@ title: Single Sign-on FAQs
 
-No. You can only configure Docker SSO to work with a single IdP. A domain can only be associated with a single IdP.Docker currently supports Azure AD and identity providers that support SAML 2.0
+You can create a test organization. Companies can set up a new five-seat Business Plan on a new organization to test with (making sure you enable SSO only, if you enforce it, all domain email users will be forced to sign in to that test tenant).
 
-### Q: Is it possible to change my identity provider after configuring SSO?
+## SAML SSO
 
-Yes. You must delete your existing IdP configuration in Docker Hub and follow the instructions to [configure SSO](index.md) using your IdP.
+### Q: Does SAML authentication require additional attributes?
 
-###  Q: Is Docker SSO available for all paid subscriptions?
+You must provide an email address as an attribute to authenticate via SAML. The ‘Name’ attribute is currently optional.
 
-Docker SSO is only available with the Docker Business subscription. [Upgrade](/subscription/upgrade/){:target="blank" rel="noopener" class=""} your existing subscription to start using Docker SSO.
+### Q: Does the application recognize the NameID/Unique Identifier in the SAMLResponse Subject?
 
-### Q: Does Docker SSO support multi-factor authentication (MFA)?
+The preferred format is your email address, which should also be your Name ID.
 
-When SSO is being used by an organization, MFA is determined at the idP level and not the Docker system.
+### Q: When SAML SSO is enforced, at what stage is the login required to be tracked through SAML? At runtime or install time?
 
-### Q: How does service accounts work with SSO?
+At runtime for Docker Desktop if it’s configured to require authentication to the organization.
+
+### Q: How long is the grace-period for using regular user id and password for the docker desktop itself regardless of the enforced SSO?
+
+Currently, we do not have a date on when the grace-period will end.
+
+### Q: Do you have any information on how to use the Docker Desktop application in accordance with the SSO users we provide? How can we verify that we are handling the licensing correctly?
+
+Verify that your users have downloaded the latest version of Docker Desktop. We will be enhancing user management observability and capabilities in the near future.
+
+## Docker org and Docker ID
+
+### Q: What’s a Docker ID? Can I retain my Docker ID when using SSO?
+
+For a personal Docker ID, a user is the account owner, it’s associated with access to the user's repositories, images, assets. An end user can choose to have a company domain email on the Docker account, when SSO is enforced, the account will be tied to the company’s organization account. Alternatively, when SSO is enforced for a company organization, any user logging in without an existing account using verified company domain email will automatically have an account provisioned, and a new Docker ID created.
```

Comment:
> when SSO is enforced for a company organization,

Should we use either company or organization here?

Reply:
Updated with organization